### ISSUE TITLE
Paget cells in skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10089]
**Preferred term label:**
Paget cells in skin

**Synonyms**

**Definition (free text, please give PubMed ID)**
Malignant, intraepithelial adenocarcinoma cells of variable sizes usually large in size, present singly or in the form of small groups within the epidermis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK563228/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000624
- label: Nipple epidermal Paget cells
- parents: [Abnormal nipple morphology (HP:0004404)](https://hpo.jax.org/app/browse/term/HP:0004404)
- pmid: PMID:33085375
- def: Malignant, intraepithelial adenocarcinoma cells of variable sizes usually large in size, present singly or in the form of small groups within the epidermis of the nipple. Cells may be ovoid, round, or signet-ring forms, usually mucin positive, and the cytoplasm may contain periodic acid-Schiff (PAS)-positive, diastase-resistant granules, indicating the presence of neutral mucopolysaccharides. The cells possess microscopic features of glandular cells with pale to clear vacuolated cytoplasm, and nuclei are usually high-grade with prominent nucleoli.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10089
- synonyms:
